ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -351,9 +382,20 @@ open class BadgeView: UIView {
 
     open override func sizeThatFits(_ size: CGSize) -> CGSize {
         let labelSize = label.sizeThatFits(CGSize(width: CGFloat.infinity, height: CGFloat.infinity))
-        let width = UIScreen.main.roundToDevicePixels(labelSize.width) + self.size.horizontalPadding * 2
+        let labelWidth = UIScreen.main.roundToDevicePixels(labelSize.width)
+        let labelHeight = UIScreen.main.roundToDevicePixels(labelSize.height)
+
+        let width: CGFloat
+        let height: CGFloat
+        if let avatarSize = dataSource?.avatar?.state.size.size {
```

Comment:
In the scenario of making this generic to a UIView instead of the Avatar, shouldn't it be safe to call [sizeToFit()](https://developer.apple.com/documentation/uikit/uiview/1622630-sizetofit) in the generic view and calculate it directly here?

Reply:
We will use `customView.bounds` before `customView.sizeThatFits(size)` to support an AvatarView with 20 X 20